From ceb4b9fc7919587de4c4c1ba6179f4d5bc8ff10e Mon Sep 17 00:00:00 2001
From: Wolfgang Merkt <w.merkt@gmail.com>
Date: Tue, 2 Aug 2022 14:22:35 +0100
Subject: [PATCH 1/2] ROS2: Set PYTHON_DEB_LAYOUT and PYTHON_STANDARD_LAYOUT

---
 CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9a1c88c..1f7bda4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -70,6 +70,10 @@ if(NOT EXISTS "${CMAKE_SOURCE_DIR}/cmake/base.cmake")
   endif()
 endif()
 
+# Required for ROS2
+set(PYTHON_DEB_LAYOUT OFF)
+set(PYTHON_STANDARD_LAYOUT ON)
+
 include("${JRL_CMAKE_MODULES}/boost.cmake")
 include("${JRL_CMAKE_MODULES}/hpp.cmake")
 include("${JRL_CMAKE_MODULES}/apple.cmake")
-- 
2.25.1

